Lib9c/Action/ActionUnAvailableException.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Runtime.Serialization;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    public class ActionUnAvailableException : InvalidOperationException
```

Comment:
How about use "Un`A`valiable" as lower case like below?

```suggestion
    public class ActionUnavailableException : InvalidOperationException
```